common/djangoapps/student/models.py

```diff
@@ -753,6 +753,17 @@ def user_post_save_callback(sender, **kwargs):
             ceas = CourseEnrollmentAllowed.for_user(user).filter(auto_enroll=True)
 
             for cea in ceas:
+                # skip enrolling already enrolled users
+                if CourseEnrollment.is_enrolled(user, cea.course_id):
+                    # Link the CEA to the user if the CEA isn't already linked to the user
+                    # (e.g. the user was invited to a course but hadn't activated the account yet)
+                    # This is to prevent students from changing e-mails and
+                    # enrolling many accounts through the same e-mail.
+                    if not cea.user:
+                        cea.user = user
+                        cea.save()
+                    continue
```

Comment:
Anyone have thoughts on doing this with `if/else` vs a `continue`? Personally think that the if/else construct conveys intention more clearly, but `continue` seems to be a more Pythonic expression?

Reply:
for example i know that if you do something like if x: return 1 else return 2 one of our python linters will complain that the else is not necessary
not sure if there's python standards for when to use continue vs if else
i don't find this continue confusing personally but you can see what others think